pkgs/development/tools/build-managers/rebar3/rebar-deps.nix

```diff
@@ -0,0 +1,58 @@
+{ fetchHex, fetchFromGitHub }:
+{
+  ssl_verify_fun = fetchHex {
+    pkg = "ssl_verify_fun";
+    version = "1.1.6";
+    sha256 = "BDB0D2471F453C88FF3908E7686F86F9BE327D065CC1EC16FA4540197EA04680";
+  };
+  relx = fetchHex {
+    pkg = "relx";
+    version = "4.3.0";
+    sha256 = "738E0949A6FC7D0DE9E4549DC0F73D9B6E05B539E1511BB248590702B3220440";
+  };
+  providers = fetchHex {
+    pkg = "providers";
+    version = "1.8.1";
+    sha256 = "E45745ADE9C476A9A469EA0840E418AB19360DC44F01A233304E118A44486BA0";
+  };
+  parse_trans = fetchHex {
+    pkg = "parse_trans";
+    version = "3.3.1";
+    sha256 = "07CD9577885F56362D414E8C4C4E6BDF10D43A8767ABB92D24CBE8B24C54888B";
+  };
+  getopt = fetchHex {
+    pkg = "getopt";
+    version = "1.0.1";
+    sha256 = "53E1AB83B9CEB65C9672D3E7A35B8092E9BDC9B3EE80721471A161C10C59959C";
+  };
+  eunit_formatters = fetchHex {
+    pkg = "eunit_formatters";
+    version = "0.5.0";
+    sha256 = "D6C8BA213424944E6E05BBC097C32001CDD0ABE3925D02454F229B20D68763C9";
+  };
+  erlware_commons = fetchHex {
+    pkg = "erlware_commons";
+    version = "1.4.0";
+    sha256 = "185ECF5CF43BAB3A013DDB3614CE7BBA7F6C7A827904E64E57DA54FCDFDCE2E6";
+  };
+  cth_readable = fetchHex {
+    pkg = "cth_readable";
+    version = "1.5.0";
+    sha256 = "BD81CDFBC9A3088621AA6AC4F20B5CEBC741D1E9C6B555F3F7D0369C9D59A87C";
```

Comment:
Please use base64 encoded hashes.

Reply:
These are generated by `rebar3nix`, however if base64 hashes are a requirement we could convert them